src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProject.cs

```diff
@@ -1126,9 +1127,48 @@ private OneOrMany<string> GetMappedAnalyzerPaths(string fullPath)
             return GetMappedRazorSourceGenerator(fullPath);
         }
 
+        if (TryRedirectAnalyzerAssembly(fullPath) is { } redirectedPath)
```

Comment:
If @jaredpar's work doesn't come first, we should then refactor this method away by converting the IsSdkRazorSourceGenerator/GetMappedRazorSourceGenerator methods into a redirector.

Reply:
Yeah, I think Jared's work is going to refactor all this and we just want to get this in so we can continue on the SDK side etc.